src/write_gcm.c

```diff
@@ -2210,12 +2210,18 @@ static monitored_resource_t *wg_monitored_resource_create_from_metadata_agent(
           "Could not create Metadata Agent URL.");
     goto error;
   }
-  char *response_buffer = NULL;
-  wg_curl_get_or_post(&response_buffer, query, body, headers, num_headers, 0);
-  monitored_resource_t *resource = parse_monitored_resource(response_buffer,
-      project_id);
-   sfree(response_buffer);
-   return resource;
+  char *response = NULL;
+  if (wg_curl_get_or_post(&response, query, body, headers, num_headers, 0)
+      != 0) {
+    ERROR("write_gcm: wg_monitored_resource_create_from_metadata_agent: "
+          "Failed to get monitored resource.");
```

Comment:
Let's change the comment to something on the lines of "Failed to get monitored resource from local metadata agent service".